xline/src/storage/kvstore.rs

```diff
@@ -182,18 +181,30 @@ impl KvStoreInner {
                 kvs.append(&mut values);
             }
             _ => {
-                let range = Range {
-                    start: req.key,
-                    end: req.range_end,
+                let range = KeyRange {
+                    start: key.to_vec(),
+                    end: range_end.to_vec(),
                 };
                 let revisions = self.index.get_range(range);
                 let mut values = self.db.get_values(&revisions);
                 kvs.append(&mut values);
             }
         }
+        kvs
+    }
+
+    #[allow(clippy::pattern_type_mismatch)]
```

Comment:
There's no match here, why do we allow this clippy?

Reply:
Removed.